### DISCUSSION TITLE
Cannot install

### DISCUSSION THREAD

**Discussion by renatobellotti** [pyvista/pyvista#1412] [Q&A] [answered]
Hi,

I've been trying to get pyvista running on my workstation (no GPU). Here is the error I get when I try to run the example code:

```python
from pyvista import examples
examples.plot_wave()
```

Here is the error message I get:

```
ERROR:root:Unable to find a valid OpenGL 3.2 or later implementation. Please update your video card driver to the latest version. If you are using Mesa please make sure you have version 11.2 or later and make sure your driver in Mesa supports OpenGL 3.2 such as llvmpipe or openswr. If you are on windows and using Microsoft remote desktop note that it only supports OpenGL 3.2 with nvidia quadro cards. You can use other remoting software such as nomachine to avoid this issue.
ERROR:root:1: #version 150
```

My employer is quite strict regarding installation of software, so I really hope this can be resolved with anaconda since I have no permissions to install anything myself.

My Python version is `3.8.8` and I'm using Jupyerlab `3.0.13` on RHEL `7.9`.

Has somebody experienced similar problems?

All the best!

**Comment by tkoyama010** [comment, marked as answer]
Please run the following code in your enviroment and paste the output below. This report helps us track down problem:

```py
# Get system info
import pyvista as pv
print(pv.Report())
```

**Reply by akaszynski** [reply]
Most likely, you're missing open gl on RHEL:
```
yum install mesa-libGL mesa-libGLU
```
If you have to do this through anaconda, try:
```
conda install -c anaconda mesa-libgl-cos6-x86_64
```

Note, you may need to extend your library path to include the anaconda library path.  See the discussion here:
https://groups.google.com/a/continuum.io/g/anaconda/c/0IqjwrUvu2U?pli=1
